src/config.ts

```diff
@@ -161,7 +161,7 @@ export interface Config {
 
   // MARK-SPECIFIC CONFIGS
   /** Area-Specific Config */
-  area?: AreaConfig;
+  area?: MarkConfig;
```

Comment:
Makes sense. This will also make it easier to switch from one mark to another without having to delete pros just to make the spec valid.